src/main/java/nl/b3p/loader/jdbc/ColumnMetadata.java

```diff
@@ -1,19 +1,7 @@
-/*
- * Copyright (C) 2017 B3Partners B.V.
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
+// Copyright (C) 2017 B3Partners B.V.
+//
+// SPDX-License-Identifier: MIT
```

Comment:
als je de repository "browsed" kun je dus niet direct zien wat de licentie van de file/broncode is, want er staat een verwijzing naar een macro voor de specifieke tooling

Reply:
Ja, dat is het idee van REUSE. Het is ook een stuk korter. Zie bijvoorbeeld een random bestand in https://github.com/torvalds/linux waar deze id's ook gebruikt worden.